aten/src/ATen/native/mkldnn/Normalization.cpp

```diff
@@ -32,10 +32,57 @@ std::tuple<Tensor, Tensor, Tensor> mkldnn_batch_norm_backward(
 
 #include <ATen/native/mkldnn/MKLDNNCommon.h>
 #include <ATen/native/mkldnn/Utils.h>
+#include <ATen/native/layer_norm.h>
 
 namespace at {
 namespace native {
 
+std::tuple<Tensor, Tensor, Tensor> mkldnn_layer_norm(
```

Comment:
we're checking for certain preconditions in fusing layer_norm within jit like optional bias, optional weight, etc. i'm a little wary of adding ops here to aten that dont cover the full semantics. i dont know if we really want to maintain layer norm in aten generally.

Reply:
That's a good point. I will still need to add some layernorm helper to this file since I can't call into MKLDNN directly from `frozen_ops_to_mkldnn`

Reply:
Let me try if I can turn this into a helper